docs/operator/configuring-serving-cr.md

```diff
@@ -0,0 +1,371 @@
+---
+title: "Configuring Serving Operator Custom Resource"
+weight: 10
+type: "docs"
+aliases:
+- /docs/operator/configuring-serving-cr/
+---
+
+# Configuring Serving Operator Custom Resource
+
+You are able to configure Knative Serving with the following options:
```

Comment:
Instead:

"The Knative Serving operator can be configured with these options:"